cli/tests/upload.rs

```diff
@@ -79,17 +77,21 @@ async fn upload_bundle() {
         })
     );
 
+    let upload_request = assert_matches!(requests_iter.next().unwrap(), RequestPayload::CreateBundleUpload(ur) => ur);
     assert_eq!(
-        requests_iter.next().unwrap(),
-        RequestPayload::CreateBundleUpload(CreateBundleUploadRequest {
-            repo: Repo {
-                host: String::from("github.com"),
-                owner: String::from("trunk-io"),
-                name: String::from("analytics-cli"),
-            },
-            org_url_slug: String::from("test-org"),
-        })
+        upload_request.repo,
+        Repo {
+            host: String::from("github.com"),
+            owner: String::from("trunk-io"),
+            name: String::from("analytics-cli"),
+        }
     );
+    assert_eq!(upload_request.org_url_slug, String::from("test-org"));
```

Comment:
optional, nit: you shouldn't need `String::from()` here, the string literal is just fine